version/version.go

```diff
@@ -18,5 +18,5 @@ package version
 
 var (
 	// Version tracks the NFD operator release
-	Version = "0.0.1"
+	Version = "0.4.0"
```

Comment:
Just a note that is this actually used somewhere? Should we drop this in `master`?

Reply:
I think ;P 0.5?